qiskit/circuit/library/standard_gates/x.py

```diff
@@ -584,8 +584,8 @@ def __init__(
 
         Args:
             label: An optional label for the gate [Default: ``None``]
-            ctrl_state: control state expressed as integer,
-                string (e.g.``'110'``), or ``None``. If ``None``, use all 1s.
+            ctrl_state: control state expressed as integer, string (e.g.``"110"``), or ``None``.
```

Comment:
urelated, just noticed the formatting issue in https://docs.quantum.ibm.com/api/qiskit/qiskit.circuit.library.C3XGate#methods

![Screenshot 2024-04-21 at 13 57 55](https://github.com/Qiskit/qiskit/assets/766693/4d1ad3e9-fdca-447b-a036-694ac4d0397c)